custom_components/ha_washdata/frontend.py

```
@@ -266,6 +269,50 @@ async def _on_lovelace_loaded(event: Event) -> None:
+    ``_register_static_path``.
+    """
+    try:
+        from homeassistant.components.http import StaticPathConfig  # pylint: disable=import-outside-toplevel
+
+        if hasattr(hass.http, "async_register_static_paths"):
+            await hass.http.async_register_static_paths(
+                [StaticPathConfig(url_path, path, cache_headers=True)]
+            )
+        else:
+            _register_static_path(hass, url_path, path)
+    except Exception as exc:  # pylint: disable=broad-exception-caught
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)